Modules/Scripted/DICOMLib/DICOMProcesses.py

```diff
@@ -344,9 +350,10 @@ def processStdoutLine(self, line):
 
 
 class DICOMSender(DICOMProcess):
-  """Code to send files to a remote host
-  (Uses storescu from dcmtk)
+  """ Code to send files to a remote host.
+      (Uses storescu from dcmtk.)
   """
+  dicomseg_altconfig_path = 'DICOM/dcmtk/storescu.cfg'
```

Comment:
Give a more meaningful name (a name that reflects why this alternative configuration exists).
Also, make it more future-proof by either using a more generic name (that allows us to add more customizations)  or by storing a list of alternative configuration filenames.

Either of these could work:

```python
extended_dicom_config_path = 'DICOM/dcmtk/storescu.cfg'
```

```python
alternative_dicom_config_paths = ['DICOM/dcmtk/storescu.cfg']
```

Reply:
I will add more documentation to the alternative configuration describing where it came from and how it can be extended. A single configuration file should be enough (hopefully) to cover all of the specialty storages covered by slicer. I'll also adopt the property name you propose in this comment.